src/storage/migrations.js

```diff
@@ -463,6 +463,16 @@ const migrationsInner: {| [string]: (LessPartialState) => LessPartialState |} =
   // Add enableReadReceipts to state.realm.
   '53': dropCache,
 
+  // Change ThemeName from 'default' or 'night' to 'light' or 'dark'
+  '54': state => ({
+    ...state,
+    settings: {
+      ...state.settings,
+      theme:
+        state.settings.theme === 'default' || state.settings.theme === 'light' ? 'light' : 'dark',
```

Comment:
I think this line should just have to be  
`state.settings.theme === 'default' ? 'light' : 'dark',`  
however when it was like this the test with the description "check dropCache at 53" in migrations-test.js (around line 141) was failing. That test's `before` and `after` block both test the `endBase`. I wasn't sure if I should have fixed that test somehow or adjusted my code to what I submitted.

Reply:
Hmm, yeah. The code should indeed look like that, so the test should be fixed to pass.

Looking back at that test, I think the issue is basically that when I wrote it (in 61c11dd5df03e5ca7bb0fefcd6e2d7b3c88fc9ce) I constructed the "before" data in a way that didn't really say the same thing as what the test case wants conceptually, and as a result it was susceptible to breaking with future changes like this one.

I just pushed to the tip of your PR branch a small commit (4247b9a39c99a4e2b50d11f1255332a68d3eb986) which should fix the issue. The fix makes sense already on top of `main`, so just rebase that commit to the start of your PR branch, and then you should be able to write the expected logic here and have things work.

Reply:
Done. I put your commit (now 16162d96e4281bf7a706fd333d6dcc25b405e805) at the beginning of this PR in the commit history and rebased my commit (b848cb68cc09ae70044a0d132bb8aab924fdd710) so it has the correct migration for the theme setting. Thanks Greg.